__tests__/utils.test.ts

```diff
@@ -23,29 +23,23 @@ afterEach(() => {
 });
 
 describe('utils', () => {
-  describe('parseQueryResult', () => {
+  describe('parseAuthenticationResult', () => {
     it('parses the query string', () => {
       expect(
-        parseQueryResult('value=test&otherValue=another-test')
+        parseAuthenticationResult('code=test&state=another-test')
       ).toMatchObject({
-        value: 'test',
-        otherValue: 'another-test'
+        code: 'test',
+        state: 'another-test'
```

Comment:
The type returned by `parseQueryResult`, only ever accounted for `code`, `state`, `error`, or `error_description`. This was testing a scenario that should not come up. So I changed it to code and state.

Probably an older test from when this code was used to parse any querystring.